src/caret_analyze/record/records_service/response_time.py

```diff
@@ -16,13 +16,16 @@
 
 from collections.abc import Iterator, Sequence
 
+from logging import getLogger
 from warnings import warn
 
 from ..column import ColumnValue
 from ..interface import RecordInterface, RecordsInterface
 from ..record_factory import RecordFactory, RecordsFactory
 from ...common import ClockConverter
 
+logger = getLogger(__name__)
```

Comment:
Is this change necessary? It seems `logger` is not used anywhere.

Reply:
Fixed in [6216774](https://github.com/tier4/caret_analyze/pull/455/commits/62167743fa765e4a6acda68e957041f46e6ff371).